akka-docs/rst/scala/stream/stream-graphs.rst

```diff
@@ -292,6 +292,9 @@ Cycles in bounded stream topologies need special considerations to avoid potenti
 This section shows several examples of problems that can arise from the presence of feedback arcs in stream processing
 graphs.
 
+In examples below runnable graphs are created but do not run because each have some issue and will not complete normally.
```

Comment:
Not sure about wording on "complete normally", perhaps call out that they deadlock?